src/index.html

```diff
@@ -7,6 +7,49 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__days">
+        <div class="calendar__days calendar__days--1">Mon</div>
+        <div class="calendar__days calendar__days--2">Tue</div>
+        <div class="calendar__days calendar__days--3">Wed</div>
+        <div class="calendar__days calendar__days--4">Thu</div>
+        <div class="calendar__days calendar__days--5">Fri</div>
+        <div class="calendar__days calendar__days--6">Sat</div>
+        <div class="calendar__days calendar__days--7">Sun</div>
+      </div>
+      <div class="calendar__month">
+        <div class="calendar__month calendar__month-length--1 calendar__month--start-days-wed"></div>
```

Comment:
here should be class to change amount of days in month from 28 to 31, u should also do it with loop